logic/src/commonMain/kotlin/com/wire/kalium/logic/data/asset/Asset.kt

```diff
@@ -48,11 +28,20 @@ enum class RetentionType {
     EXPIRING
 }
 
-sealed class AssetType(open val name: String)
-data class FileAsset(val mimetype: String) : AssetType(mimetype)
-sealed class ImageAsset(override val name: String) : AssetType(name) {
-    object JPEG : ImageAsset(name = "image/jpeg")
-    object PNG : ImageAsset(name = "image/png")
+sealed class AssetType(open val mimeType: String)
+data class FileAsset(val fileExtension: String) : AssetType("file/$fileExtension")
+sealed class ImageAsset(override val mimeType: String) : AssetType(mimeType) {
+    object JPEG : ImageAsset(mimeType = "image/jpeg")
+    object JPG : ImageAsset(mimeType = "image/jpg")
+    object PNG : ImageAsset(mimeType = "image/png")
 }
 
+fun isValidImage(mimeType: String): Boolean =
```

Comment:
Should we support also HEIC for iOS, because I don't have full context here 😅

Reply:
I added the heic as valid inline images. Only problem there still is that web client doesn't seem to like these images format and doesn't know how to render them correctly.